### PR TITLE
Add bookstack example

### PR DESCRIPTION
**Pull request by Sblop**
I have added lldap to bookstack. Although very easy, I thought it appropriate too add the example here 
/B

### CODE REVIEW

File: example_configs/bookstack.env.example [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# If using LDAP over SSL you should also define the protocol:
+# LDAP_SERVER=ldaps://example.com:636
+
+# The base DN from where users will be dk within
+LDAP_BASE_DN=ou=people,dc=example,dc=com
+
+# The full DN and password of the user used to search the server
+# Can both be left as false to bind anonymously
+LDAP_DN=cn=admin,ou=people,dc=example,dc=com
+LDAP_PASS=YOUR-ADNUB-PASSWORD-HERE
```
Comment (nitnelave) [inline example_configs/bookstack.env.example:19]: typo: ADMIN

**Review by nitnelave — CHANGES_REQUESTED** [review]
Thanks! Could you add a link to it in the readme? https://github.com/nitnelave/lldap#sample-client-configurations

**Review by Sblop — COMMENTED** [review]
Now added to example liste. Corrected admin spelling mistake.

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Thanks a lot! I'll wait until the checks pass, then I'll merge this.

**Pull request merged**